pages/index.js

```diff
@@ -1,20 +1,25 @@
 import PetList from "@/components/PetList";
 import StyledHeading from "@/components/Styles/StyledHeading";
 import PetForm from "@/components/PetForm";
-import { useState } from "react";
 import CurrentPet from "@/components/CurrentPet";
 import CurrentPetStats from "@/components/CurrentPetStats";
 import EditForm from "@/components/EditForm";
 import EliminateForm from "@/components/EliminateForm";
 import TombstoneButton from "@/components/TombstoneButton";
 import StyledSection from "@/components/Styles/StyledSection";
 import styled from "styled-components";
+import InteractionMenu from "@/components/InteractionMenu";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function HomePage() {
-  const [selectedPet, setSelectedPet] = useState();
-  const [mode, setMode] = useState("select");
+  const [selectedPet, setSelectedPet] = useLocalStorageState("selectedPet", {
+    defaultValue: {},
+  });
+  const [mode, setMode] = useLocalStorageState("mode", {
+    defaultValue: "select",
+  });
 
-  let isDead = false;
+  let isDead = selectedPet.health === 0;
```

Comment:
```suggestion
const isDead = selectedPet.health === 0;
```

You should not redeclare variables like this in react. You then need to find another mechaism to set the selectedPet.health to 0